database/firebird/firebird_test.go

```diff
@@ -0,0 +1,197 @@
+package firebird
+
+import (
+	"context"
+	"database/sql"
+	sqldriver "database/sql/driver"
+	"fmt"
+	"github.com/golang-migrate/migrate/v4"
+	"io"
+	"strings"
+	"testing"
+
+	"github.com/dhui/dktest"
+
+	dt "github.com/golang-migrate/migrate/v4/database/testing"
+	"github.com/golang-migrate/migrate/v4/dktesting"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+
+	_ "github.com/nakagami/firebirdsql"
+)
+
+const (
+	user     = "test_user"
+	password = "123456"
+	dbName   = "test.fdb"
+)
+
+var (
+	opts = dktest.Options{
+		PortRequired: true,
+		ReadyFunc:    isReady,
+		Env: map[string]string{
+			"FIREBIRD_DATABASE": dbName,
+			"FIREBIRD_USER":     user,
+			"FIREBIRD_PASSWORD": password,
+		},
+	}
+	specs = []dktesting.ContainerSpec{
+		{ImageName: "jacobalberty/firebird", Options: opts},
```

Comment:
Specify versions that are supported. e.g. `3.0` and `2.5-ss`

Reply:
Added 3.0 and 2.5-ss